daemon/daemon/loadbalancer.go

```diff
@@ -335,6 +335,36 @@ func (d *Daemon) RevNATDump() ([]types.L3n4AddrID, error) {
 	return dump, nil
 }
 
+// WrrDump dumps a DeepCopy of cilium's wrr sequences.
+func (d *Daemon) WRRDump() ([]lbmap.ServiceRR, error) {
+	dump := []lbmap.ServiceRR{}
+
+	d.loadBalancer.BPFMapMU.RLock()
+	defer d.loadBalancer.BPFMapMU.RUnlock()
+
+	for _, u := range d.loadBalancer.SVCMap {
+		weights := []uint16{}
+		sum := uint16(0)
+		for _, v := range u.BES {
+			weights = append(weights, v.Weight)
+			sum += v.Weight
+		}
+		if sum == 0 {
+			continue
+		}
+		seq := lbmap.GenerateWrrSeq(weights)
+		if seq == nil {
+			return nil, fmt.Errorf("unable to generate weighted round robin seq for %+v with value %+v", u.FE, weights)
```

Comment:
I think the FE has a String() method. If it has it please us it, along with "%s", for prettier output